packages/data/src/abbreviations.ts

```diff
@@ -21,7 +21,7 @@ export const CHOP_MOVED_NOTES = [
 export const FINESSED_NOTES = ["f", "hf", "pf", "gd", "utf"];
 export const NEEDS_FIX_NOTES = ["fix", "fixme", "needs fix"];
 export const BLANK_NOTES = ["blank"];
-export const CLUED_NOTES = ["clued"];
+export const CLUED_NOTES = ["clued", "y"];
```

Comment:
y is short for yellow so this seems like a bug?

Reply:
Convenience? :)